f/connectors/alerts/alerts_gcs.py

```diff
@@ -480,116 +546,137 @@ def handle_output(self, geojsons, metadatas):
+                                satellite,
+                                territory_id,
+                                territory_name,
+                                year_detec,
+                                source,
+                                g__type,
+                                g__coordinates,
+                                length_alert_km,
+                                alert_source,
+                            ]
+
+                            self._safe_insert(cursor, table_name, columns, values)
 
                         except errors.UniqueViolation:
                             logger.info(
-                                f"Skipping insert due to UniqueViolation, this alert has been processed already in the past: {source}"
+                                f"Skipping insert due to UniqueViolation, this entry has been processed already in the past: {_id}"
                             )
                             continue
                         except Exception:
                             logger.exception(
                                 "An unexpected error occurred while processing feature"
                             )
                             raise
-                    # End of inner loop, commit the transaction after processing all features
                     conn.commit()
-                except Exception:
-                    logger.exception("An error occurred while processing GeoJSON")
-                    conn.rollback()  # Rollback the transaction in case of an error
+                except Exception as e:
+                    logger.exception(
+                        f"An error occurred while processing alerts: {str(e)}"
+                    )
+                    conn.rollback()
                     raise
 
-            if metadatas:
-                self._create_metadata_cols(metadatas[0], table_name)
+            if alerts_metadata:
+                self._create_alerts_metadata_table(cursor, table_name)
             else:
                 logger.info("No alerts metadata to store.")
 
-            for metadata in metadatas:
+            for metadata in alerts_metadata:
                 try:
-                    cursor.execute("BEGIN")  # Start a new transaction
-                    source = metadata["source"]
+                    cursor.execute("BEGIN")
 
+                    _id = metadata.get("metadata_uuid")
                     territory_id = metadata.get("territory_id")
                     type_alert = metadata.get("type_alert")
                     month = metadata.get("month")
                     year = metadata.get("year")
                     total_alerts = metadata.get("total_alerts")
                     description_alerts = metadata.get("description_alerts")
                     confidence = metadata.get("confidence")
-                    metadata_uuid = metadata.get("metadata_uuid")
-                    source = metadata.get("source")
```

Comment:
FYI: this column was storing the name of the CSV e.g. `alerts_history.csv`. I don't think it's necessary to include this.